app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -739,59 +748,121 @@ export default class SampleDetails extends React.Component {
     )
   }
 
-  addManualCas(e) {
-    DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // addManualCas(e) {
```

Comment:
Ideally remove code that's commented out.

Reply:
done